modules/using-an-algorithm/pages/index.adoc

```diff
@@ -39,7 +166,7 @@ If your query requires a UDF, ensure that the xref:gsql-ref:querying:func/query-
 
 If your query does not require a UDF, you can proceed to the next step.
 
-== Create query
+=== Create query
 If the algorithm you wish to use is not yet installed in your TigerGraph instance, you need to xref:gsql-ref:querying:query-operations.adoc#_create_query[create] and xref:gsql-ref:querying:query-operations.adoc#_install_query[install it].
```

Comment:
If the algorithm you want is not yet installed in your TigerGraph instance, and if you do not use the GraphStudio simplified installation process, then you can install the algorithm as you would add any query to the database.
Perform xref:gsql-ref:querying:query-operations.adoc#_create_query[create] and xref:gsql-ref:querying:query-operations.adoc#_install_query[install it].